modules/ROOT/pages/microprofile-config-properties.adoc

```diff
@@ -251,10 +251,16 @@ The following MicroProfile Config properties can be specified when MicroProfile
 |`mp.metrics.appName`
 |Passes the value of the  `_app` tag from the application server to the application. This property is used to avoid metric name collisions when multiple applications are deployed to the runtime. It is defined through a `microprofile-config.properties` file in the `META-INF` directory of the application. The value of this tag is appended to the output of each metric that originates from that application.
 
-In MicroProfile 5.0, the  `_app` tag that was defined in prior releases by the `mp.metrics.appName` MicroProfile Config property is designated as `mp_app`.
+In MicroProfile Metrics 5.0, the  `_app` tag that was defined in prior releases by the `mp.metrics.appName` MicroProfile Config property is designated as `mp_app`. When running in an MicroProfile Metrics 5.x environment, if the `mp.metrics.appName` property is defined, the `mp.metrics.defaultAppName` property must be defined as well.
```

Comment:
They will be different values. 
A vendor metric (provided by Open Liberty) can originate from the server side and application side.
For example, servlet metrics.


If the application created a servlet metric it can look like this:
`servlet_request_total{mp_scope="vendor",servlet="TestAppJakarta_sand_box_MyApplication",} 1.0`
The servlet created on the server side (i.e., hitting the `/metrics` endpoint can generate one will create one like this:
`servlet_request_total{mp_scope="vendor",servlet="io_openliberty_microprofile_metrics_5_0_public_internal_PublicMetricsRESTProxyServlet",} 4.0`

The tag set consists of `mp_scope` and `servlet`.


If the `mp.metrics.appName` is defined by an application (we'll give it a value of `myApp` it can create a metric like this:
`servlet_request_total{mp_app="myApp",mp_scope="vendor",servlet="TestAppJakarta_sand_box_MyApplication",} 1.0`

The servlet created on the server side (i.e., hitting the `/metrics` endpoint can generate one will create one like this:
`servlet_request_total{mp_scope="vendor",servlet="io_openliberty_microprofile_metrics_5_0_public_internal_PublicMetricsRESTProxyServlet",} 4.0`

At this point, the server side metric is missing the `mp_app`. This causes a mismatch in tags and will cause a registration error for the second metric to be registered.

The `mp.metrics.defaultAppName` serves to provide this `mp_app` tag for server side metrics (it can be any value they wish and does not have to match).

Reply:
Perhaps:

In MicroProfile Metrics 5.0 and later, the  `_app` tag that is defined in prior releases by the `mp.metrics.appName` MicroProfile Config property is designated as `mp_app`. In a MicroProfile Metrics 5.x environment, if the `mp.metrics.appName` property is defined, you must also define the `mp.metrics.defaultAppName` property. See the `mp.metrics.defaultAppName` for more information.

Reply:
Thanks- updated my suggestion.